AsyncDisplayKit/Details/ASCollectionViewFlowLayoutInspector.m

```diff
@@ -79,6 +79,12 @@ - (ASSizeRange)collectionView:(ASCollectionView *)collectionView constrainedSize
   return NodeConstrainedSizeForScrollDirection(collectionView);
 }
 
+- (ASScrollDirection)scrollableDirections
```

Comment:
I see, this is actually in ASCollectionViewLayoutInspector (not Flow) — the file just happens to contain both objects, and is named after the Flow variant instead of the more generic name (kinda weird, but predates this diff).